internal/handlers/handler_test.go

```diff
@@ -498,6 +498,42 @@ func TestReadOnlyHandlers(t *testing.T) {
 			),
 		},
 
+		"FindProjectionActorsFirsAndLastName": {
+			req: types.MustMakeDocument(
+				"find", "actor",
+				"projection", types.MustMakeDocument(
+					"first_name", int32(1),
+					"last_name", int32(1),
+				),
+				"filter", types.MustMakeDocument(
+					"actor_id", int32(28),
+				),
+			),
+			reqSetDB: true,
+			resp: types.MustMakeDocument(
+				"cursor", types.MustMakeDocument(
+					"firstBatch", types.MustNewArray(
+						types.MustMakeDocument(
+							"first_name", "WOODY",
+							"last_name", "HOFFMAN",
+						),
+					),
+					"id", int64(0),
+					"ns", "pagila.actor",
```

Comment:
I think it might be a bit more clear to set it to empty string there, and patch for both monila and pagila in compareFunc

Reply:
Good point, fixed. Thanks!